typescript/sdk/src/ism/EvmIsmModule.hardhat-test.ts

```diff
@@ -65,10 +73,21 @@ const randomIsmConfig = (depth = 0, maxDepth = 2): IsmConfig => {
       return config;
     }
     case ModuleType.AGGREGATION: {
-      const n = randomInt(5, 1);
-      const modules = new Array<number>(n)
-        .fill(0)
-        .map(() => randomIsmConfig(depth + 1));
+      const n = randomInt(2, 1);
+      const moduleTypes = new Set();
+      const modules = new Array<number>(n).fill(0).map(() => {
+        let moduleConfig;
+        let moduleType;
+
+        // Ensure that we do not add the same module type more than once per level
+        do {
+          moduleConfig = randomIsmConfig(depth + 1, maxDepth);
+          moduleType = moduleConfig.type;
+        } while (moduleTypes.has(moduleType));
+
+        moduleTypes.add(moduleType);
+        return moduleConfig;
```

Comment:
maybe `moduleType` could be an optional arg for `randomIsmConfig`, so we could iterate through each moduletype instead of the do-while